Android/app/src/main/java/io/github/project_travel_mate/destinations/funfacts/FunfactFragment.java

```diff
@@ -75,26 +75,37 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
         if (fact != null) {
             (holder.desc).setText(fact.getText());
             (holder.title).setText(fact.getTitle());
+            (holder.source).setText(fact.getSource());
```

Comment:
Add a check before this line, 
if (fact.getSource() != null && fact.getSource() != "null")